src/obu.c

```diff
@@ -41,6 +41,10 @@
 
 #if defined(AVIF_CODEC_AVM)
 #include "config/aom_config.h"
+#if CONFIG_LR_IMPROVEMENTS
+#include "av1/common/enums.h"
+#include "av1/common/restoration.h"
```

Comment:
These two headers are internal headers. We should avoid including them. At least add a TODO comment.